### PR TITLE
Specific projects within each branches of an organisation

### PR DESCRIPTION
**Pull request by ibramoha**

### CODE REVIEW

File: frontend/src/actions/getApiData.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
 import axios from 'axios';
 import { LIST_OF_CATEGORIES, LIST_OF_BOROUGHS, LIST_OF_AREAS, FILTERED_BRANCHS_BY_CATEGORY } from './types';
 
-const api = 'http://localhost:3002' || process.env.DATABASE_URL;
+const api = 'http://localhost:3002/api' || process.env.DATABASE_URL;
```
Comment (sentayhuJo) [inline frontend/src/actions/getApiData.js]: check `Mohameds` work , talk to him , he will show you

File: backend/db/seeds/development/shows_seed.js [hunk 2]
```diff
@@ -11,37 +11,43 @@ const loadSeedToDb = async () => {
     const d = {
       org_name: row.name || 'not provided',
       website: row.branches[0].Website || 'not provided',
-      branch: row.branches.map(branch => (
-        {
-          borough: branch.Borough || 'not provided',
-          service: [{
+      branch: row.branches.map(branch => ({
+        borough: branch.Borough || 'not provided',
+        project: 'not provided',
```
Comment (sentayhuJo) [inline backend/db/seeds/development/shows_seed.js]: where is `tag`

Comment (ibramoha) [reply backend/db/seeds/development/shows_seed.js]: @sentayhuJo Done!

File: frontend/src/actions/getApiData.js (hunk 1 above)
Comment (ibramoha) [reply frontend/src/actions/getApiData.js]: Done.

**Comment by ibramoha** [comment]
@kkarimi done!

**Comment by ibramoha** [comment]
It is resolved @kkarimi

**Pull request merged**